internal/controlplane/handlers_repositories.go

```diff
@@ -445,3 +450,40 @@ func (s *Server) ListRemoteRepositoriesFromProvider(
 
 	return out, nil
 }
+
+func deleteRepositoryAndWebhook(
+	s *Server,
```

Comment:
Should this not rather be defined as a method of `Server`?

Reply:
Yes, it should be. Done.